api/src/opentrons/protocol_engine/state/addressable_areas.py

```diff
@@ -462,6 +470,30 @@ def get_addressable_area_center(self, addressable_area_name: str) -> Point:
             z=position.z,
         )
 
+    def get_fixture_by_deck_slot_name(
+        self, slot_name: DeckSlotName
+    ) -> Optional[CutoutFixture]:
+        """Get the Cutout Fixture of a fixture currently loaded into a specific Deck Slot."""
```

Comment:
Terminology thing: 
```suggestion
        """Get the Cutout Fixture currently loaded where a specific Deck Slot would be."""
```

* Fixtures are not loaded into deck slots; deck slots are a kind of fixture, and fixtures are loaded into cutouts.
* "Cutout Fixture of a fixture" seems like it was just a typo?

Reply:
Just a nomenclature typo, thanks for the catch.